docs/snippets/angular/list-story-reuse-data.ts.mdx

```diff
@@ -6,9 +6,9 @@ import * as ListItemStories from './ListItem.stories';
 export const ManyItems = Template.bind({});
 ManyItems.args = {
   items: [
-    ...ListItemStories.Selected.args,
-    ...ListItemStories.Unselected.args,
-    ...ListItemStoriesUnselected.args,
+    { ...ListItemStories.Selected.args },
+    { ...ListItemStories.Unselected.args },
+    { ...ListItemStories.Unselected.args },
```

Comment:
I don't understand this change in the context of the diff. Was is just completely wrong in the past?

Reply:
@phated yes it was i'm fixing it in the react snippets as i type this. If you only leave `...ListItemStories.Unselected.args` as opposed to the way i committed it, it will yield two things:
- The story will not show.
- If you check the dev console you'll see the warning pop up mentioning  something like "The prop x is not iterable...Symbol.Iterator".